File: BitPlusPlus.cpp
#include <bits/stdc++.h>
using namespace std;

int main(void) {
    int x;
    cin >> x;
    vector<string> statement;
    string s;
    for (int i=0; i<x; i++) {
        cin >> s;
        statement.push_back(s);
    }
    int result=0;
    for (auto i=statement.begin(); i!=statement.end(); i++) {
        if ((*i == "X++") || (*i == "++X")) 
            result++;
        if((*i == "X--") || (*i == "--X"))
            result--;
    }
    cout << result << endl;

    return 0;
}

File: Python/BitPlusPlus.py
def main():
    x = int(input())
    statement = [input() for i in range(x)]
    result = 0
    for i in range(len(statement)):
        if (i == "X++") or (i == "++X"):
            result += 1
        elif (i == "X++") or (i == "++X"):
            result -= 1
    print(result)


if __name__ == "__main__":
    main()
